HideAndSeek.js

```diff
@@ -58,7 +58,7 @@ function clearPr() {
 
 //return the number for the pull request
 function getPullRequest() {
-  
+  $(".gh-header-number").innerText
```

Comment:
Maybe rename the method to getPullRequestNumber?

Reply:
fixed https://github.com/kyleroush/PR-Hide-And-Seek/pull/2/files#diff-f3436ee3bbe96e2ed643a9f81611910eR60